packages/node/src/cron/index.ts

```diff
@@ -0,0 +1,8 @@
+import { instrumentNodeCron } from './node-cron';
+
+/**
+ * Methods to instrument cron libraries for Sentry check-ins.
+ */
+export const cron = {
+  instrumentNodeCron,
```

Comment:
Can we get this rid of this barrel file and just move this to `src/index.ts`? It makes it really hard to trace.